python/pyproject.toml

```diff
@@ -53,6 +53,7 @@ Pillow = {version = "^9.2.0", optional = true}
 
 # Fugue related dependencies
 fugue = {version = "^0.7.2", optional = true}
+requests = "^2.28.1"
```

Comment:
this is forcing people to update their version of requests to a build that was released on June 29, 2022 or later, do we need to do be that strict? If a user has an older version of requests already, this entry says whylogs won't work unless they update, but that is probably not true.

I'd suggest change the version number here to allow at least as old as requests 2.27.0 since that still supported python 3.6 and has python 3.10 support, but probably we could support very old versions as we aren't using advanced features of this dependency:
https://requests.readthedocs.io/en/latest/community/updates/#id4

Reply:
Thanks!